### ISSUE TITLE
Unlabeled button in UI

### ISSUE THREAD

**Issue by RealAmethyst** [jellyfin/jellyfin-web#7457]
### Description of the bug

When clicking into a show you have buttons like mark as watched, ad to favorites and other stuff, one button there does not have a label. It is the button right before mark as watched. 

<img width="2560" height="1392" alt="Image" src="https://github.com/user-attachments/assets/5636fbd3-7c5f-4560-b156-7864c10a9026" />

### Reproduction steps

Use a screen reader to validate this, navigate to the screen shown above and make your screen reader focus on the button, it should just speak button and nothing more.

### What is the current _bug_ behavior?

Screen readers only say "button" meaning I have no idea of knowing what it is.

### What is the expected _correct_ behavior?

It should speak out what this button does.

### Jellyfin Server version

10.11.5

### Specify the build version

latest

### Environment

```markdown
- OS:win11
- Linux Kernel:
- Virtualization:
- Clients:
- Browser:chrome
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- CPU Model:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Jellyfin Data Storage & Filesystem:
- Media Storage & Filesystem:
- External Integrations:
```

### Jellyfin logs

```shell
no relevant logs
```

### FFmpeg logs

```shell

```

**Comment by geebru** [comment]
I believe the button in question is the Play button on a playable card:

<img width="398" height="310" alt="Screenshot of a show episode being hovered, showing the Play button in the center of the card" src="https://github.com/user-attachments/assets/098f697d-739d-4675-b611-3801fd5ec8db" />

And it does appear to be unlabeled entirely as it's missing hidden accessible text (preferred) or a `title`:
```html
<button is="paper-icon-button-light" class="cardOverlayButton cardOverlayButton-hover itemAction paper-icon-button-light cardOverlayFab-primary" data-action="resume">
    <span class="material-icons cardOverlayButtonIcon cardOverlayButtonIcon-hover play_arrow" aria-hidden="true"></span>
</button>
```

**Comment by theguymadmax** [comment]
Fixed by #7514
